modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+// CNF-950 4.7 Provisioning and deploying a Distributed Unit (DU) manually
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-provisioning-deploying-a-distributed-unit-manually_{context}"]
+= Provisioning and deploying a Distributed Unit (DU) manually
+
+Radio access network (RAN) is composed of Centralized Units (CU), Distributed Units (DU) and Radio Units (RU).
+RAN from the telecommunications standard perspective is shown below:
+
+image::135_OpenShift_Distributed_Unit.svg[High level RAN overview]
+
+From the three components composing RAN, the CU and DU can be virtualized and implemented as cloud-native functions.
+
+The CU and DU split architecture is driven by real-time computing and networking requirements.
```

Comment:
Again, each of these sentences in on a new line. The should probably  just flow together using soft wrap.

Reply:
Approved.